rosidl_runtime_rs/src/sequence.rs

```diff
@@ -463,14 +511,14 @@ impl<T: SequenceAlloc> Iterator for SequenceIterator<T> {
         self.idx += 1;
         Some(elem)
     }
-
+    /// returns the remaining length of the sequence
```

Comment:
This comment doesn't really explain much. Here is some inspiration from the iterator trait
https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.size_hint

Also, this is not something to be addressed in this PR but I wonder why Sequence doesn't actually implement iterator.